docarray/computation/torch_backend.py

```diff
@@ -64,6 +65,14 @@ def to_device(
     def n_dim(array: 'torch.Tensor') -> int:
         return array.ndim
 
+    @staticmethod
+    def shape(array: 'torch.Tensor') -> Tuple:
+        return array.size()
```

Comment:
I think torch.size return a different object that np.shape. Probably array.shape is better here

Reply:
okay yeah you are right my bad. As you want then